services/venice-controller/src/main/java/com/linkedin/venice/controller/Admin.java

```diff
@@ -225,7 +259,8 @@ Version incrementVersionIdempotent(
       Optional<X509Certificate> requesterCert,
       long rewindTimeInSecondsOverride,
       Optional<String> emergencySourceRegion,
-      boolean versionSwapDeferred);
+      boolean versionSwapDeferred,
+      String targetedRegions);
```

Comment:
Personally, I would prefer to do the string parsing outside the `incrementVersionIdempotent` and pass a list/set of regions to it to keep the API implementation simple.

Reply:
Let's keep it same way as regionFilter if you have no strong objection